src/Workspaces/Core/Portable/Options/EditorConfig/EditorConfigStorageLocation.cs

```diff
@@ -13,11 +13,13 @@ public static EditorConfigStorageLocation<bool> ForBoolOption(string keyName)
         public static EditorConfigStorageLocation<int> ForInt32Option(string keyName)
             => new EditorConfigStorageLocation<int>(keyName, s_parseInt32, s_getInt32EditorConfigStringForValue);
 
+#if !CODE_STYLE
         public static EditorConfigStorageLocation<CodeStyleOption<bool>> ForBoolCodeStyleOption(string keyName)
             => new EditorConfigStorageLocation<CodeStyleOption<bool>>(keyName, s_parseBoolCodeStyleOption, s_getBoolCodeStyleOptionEditorConfigStringForValue);
 
         public static EditorConfigStorageLocation<CodeStyleOption<string>> ForStringCodeStyleOption(string keyName)
             => new EditorConfigStorageLocation<CodeStyleOption<string>>(keyName, s_parseStringCodeStyleOption, s_getStringCodeStyleOptionEditorConfigStringForValue);
+#endif
```

Comment:
why does this need to be ifdef'ed out for CODE_STYLE but, say, the ForInt32Option code doesn't need to be?

Reply:
I didn't bring `CodeStyleOption<T>` down to the code style layer. I actually wanted to avoid all of the `IOption` related code but it would have made the change much more complicated in the features layer.